Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -1008,7 +1008,7 @@
     <string name="submission_qr_code_scan_invalid_dialog_button_negative">"Cancel"</string>
 
     <!-- QR Code Scan Screen -->
-    <string name="qr_code_scan_title">"QR-Code Scan"</string>
+    <string name="qr_code_scan_title">"QR Code Scan"</string>
     <!-- YTXT: instruction text for QR code scanning  -->
     <string name="qr_code_scan_body">"Position the QR code in the frame."</string>
 
```

Comment:
you can probably also delete the lines below now

Reply:
https://github.com/corona-warn-app/cwa-app-android/pull/2723/files/1e6a6f878530687cb3abed33ef5d20899b0645fd#diff-4a9d100d540b5bebcf8f85484c136dff32ae062cb2bebea6a3a1c8d17b1f5f74R1015-R1017